README.md

```diff
@@ -1,59 +1,111 @@
 # drupal-check [![Build Status](https://travis-ci.com/mglaman/drupal-check.svg?branch=master)](https://travis-ci.com/mglaman/drupal-check) [![Latest release](https://img.shields.io/github/release/mglaman/drupal-check.svg)](https://github.com/mglaman/drupal-check/releases/latest)
 
+Built on [phpstan](https://github.com/phpstan/phpstan), this static analysis tool will check for correctness (e.g. using a class that doesn't exist), deprecation errors, and more.
+
+Why? While there are many static analysis tools out there, none of them run with the Drupal context in mind. This allows checking contrib modules for deprecation errors thrown by core.
+
+Are you ready for Drupal 9? Check out our [Drupal 9 Readiness](https://github.com/mglaman/drupal-check/wiki/Drupal-9-Readiness) instructions for details on how this tool can help.
+
 ## Requirements
 
-You must have PHP 7.1 or greater, which is a requirement of PHPStan, the static analyzer backing this tool.
+* PHP >=7.1
 
-## Usage
+## Installation
+
+The easiest way to install is by downloading the latest PHAR and putting it into your path. For example:
```

Comment:
Let's say preferred as well

Reply:
I'm going to skip this because I suspect this wouldn't be broadly held -- I'll include it if you insist, though.

I think the case for using composer (global or local) is valid, and is probably the case that I personally would use (especially because I want to make sure I have the latest release; if I'm pasting a curl command I may have an old version).

Reply:
True. The phar does not have auto updating available.